js/stdlib/ts/shared_impl.ts

```diff
@@ -593,6 +593,10 @@ export type ISimTxn<Token, ContractInfo> = {
 } | {
   kind: 'tokenDestroy',
   tok: Token,
+} | {
+  kind: 'tokenAccepted',
```

Comment:
You need to increment the JS version number in JS.hs and in the two connectors because of this